apps/server/src/modules/board/service/drawing-element-adapter.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common';
+import { LegacyLogger } from '@src/core/logger';
+import { firstValueFrom } from 'rxjs';
+import { Configuration } from '@hpi-schul-cloud/commons/lib';
+import { HttpService } from '@nestjs/axios';
+
+@Injectable()
+export class DrawingElementAdapterService {
```

Comment:
Please move this out of the board module, probably into a "tldraw-client" module, analog to the filestorage-client module.